docs/source/reexecuting_updating_experiments.rst

```diff
@@ -0,0 +1,134 @@
+Re-executing and updating experiments
+=====================================
+
+*This tutorial covers updating and re-executing individual cases in an experiment.*
+
+Filtered case execution
+***********************
+
+It is possible to update and run a subset of cases in a batch simulation. Call the execute method on the 
+experiment object with the ``with_cases`` argument set to a list of the cases that should be executed. 
+Observe that if the experiment has not been executed yet it will not have any cases that we can access. 
+To work around this we can first execute the experiment and specify that no cases should be executed::
+
+    experiment_expand = experiment.execute(with_cases=[]).wait()
+
+This generate all the cases for a given experiment definition and compiles all FMUs needed for running the cases. 
+But, no cases are executed when the ``execute`` method is called with an empty list argument.
+
+Now, to execute only a subset of cases (say "case_2" and "case_3") in a batch simulation with 20 cases, we can 
+just pass the case objects as a list to the ``with_cases`` argument::
+   
+   case_2 = experiment.get_case('case_2')
+   case_3 = experiment.get_case('case_3')
+
+   experiment = experiment.execute(with_cases=[case_2, case_3]).wait()
+
+Updating cases
+**************
+
+The case input for a specific case in a batch simulation can be updated and the case can be re-executed.
+The following input values could be updated.
+1. Custom function parameters - E.g. - 'start_time', 'final_time' for a dynamic simulation.
+2. Simulation and solver options - E.g. - 'ncp', 'atol' etc for a dynamic simulation.
+3. Non-structural parameters - E.g. - 'PI.k' for the Modelica.Blocks.Examples.PID_Controller.
```

Comment:
This list is not formatted correctly when I build HTML. Maybe we need an empty line before it?